src/main/java/com/unosquare/cvgenerator/service/GreetingServiceImpl.java

```diff
@@ -16,16 +17,17 @@
 public class GreetingServiceImpl implements GreetingService {
 
     private final HelloRepository helloRepository;
+    private final ModelMapper modelMapper;
 
     @Override
-    public GreetingModelView findById(int id) {
+    public GreetingDTO findById(Integer id) {
         Optional<Greeting> result = helloRepository.findById(id);
         if (!result.isPresent()) {
-            throw new RuntimeException(String.format("Cannot find object with id %s", id));
+            String error = String.format("Cannot find object with id %s", id);
+            log.error(error);
+            throw new IdDoesNotExistException(error);
```

Comment:
Something that would be great to see in future would be a `@ControllerAdvice` exception handler - something we add in another ticket, allows you to handle your exceptions in the one place without worrying about handling exceptions in your business logic if you don't need to (e.g overuse of try catch/throws method signature)

Reply:
Agreed Controller advice for the IdDoesnotExistException would work well here.